src/helpers/form-validator.js

```diff
@@ -0,0 +1,54 @@
+import { TEXT_MIN_LENGTH, URL_REGEX, EMAIL_REGEX } from 'constants.js';
+
+class FormValidator {
+
+  validate = (attributeData) => {
+    let validations = attributeData['validations']
+    return validations.map((validationKey) => {
+      switch(validationKey) {
+        case 'presence':
+          return this.validatePresence(attributeData.value);
+        case 'email':
+          return this.validateEmail(attributeData.value);
+        case 'url':
+          return this.validateURL(attributeData.value);
+        case 'min_length':
+          return this.validateLength(attributeData.value);
+        case 'acceptance':
+          return this.validateAcceptance(attributeData.checked);
+      }
+    }).filter(Boolean);
```

Comment:
Let's not filter based on if it's returning some string or not, but through something like 
`filter(({ error }) => error)`